src/protectionmarketlist.schema.json

```diff
@@ -0,0 +1,447 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Protection Market List",
+  "description": "Schema for lists of markets compatible with the Cozy Protocol",
```

Comment:
Wonder if we should say "markets *available on* the Cozy Protocol" instead of *compatible with*. My thinking is this is slightly more accurate, since any market that would be on this list would have to be deployed *through* the protocol. "compatible with" makes it sound like something that could have been deployed separate to the protocol but is useable with it through conformity to a standard or something.

Reply:
I like this, changed in 9dfee95d9da8dc8c38890621372e465b3e83d346